<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ColorListWindow</class>
 <widget class="QDockWidget" name="ColorListWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1123</width>
    <height>721</height>
   </rect>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <property name="windowTitle">
    <string>Dialog</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_left">
      <item>
       <widget class="QTableView" name="tableView">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="removeButton">
          <property name="text">
           <string>Remove</string>
          </property>
          <property name="autoDefault">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="upButton">
          <property name="toolTip">
           <string>Move selected item up</string>
          </property>
          <property name="text">
           <string>Move up</string>
          </property>
          <property name="icon">
           <iconset resource="../resources.qrc">
            <normaloff>:/icons/icons/baseline-keyboard_arrow_up-24px.svg</normaloff>:/icons/icons/baseline-keyboard_arrow_up-24px.svg</iconset>
          </property>
          <property name="arrowType">
           <enum>Qt::NoArrow</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="downButton">
          <property name="toolTip">
           <string>Move selected item donw</string>
          </property>
          <property name="text">
           <string>Move down</string>
          </property>
          <property name="icon">
           <iconset resource="../resources.qrc">
            <normaloff>:/icons/icons/baseline-keyboard_arrow_down-24px.svg</normaloff>:/icons/icons/baseline-keyboard_arrow_down-24px.svg</iconset>
          </property>
          <property name="arrowType">
           <enum>Qt::NoArrow</enum>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="newButton">
          <property name="text">
           <string>New</string>
          </property>
          <property name="autoDefault">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_right">
      <property name="sizeConstraint">
       <enum>QLayout::SetMinimumSize</enum>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>10</number>
      </property>
      <item>
       <widget class="color_widgets::ColorWheel" name="wheel" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>180</width>
          <height>180</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tabRGB">
         <attribute name="title">
          <string>RGB</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout">
          <item row="2" column="0">
           <widget class="QLabel" name="label_red">
            <property name="text">
             <string>Red</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QSpinBox" name="spin_red">
            <property name="maximum">
             <number>255</number>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_green">
            <property name="text">
             <string>Green</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="color_widgets::GradientSlider" name="slider_green">
            <property name="maximum">
             <number>255</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="QSpinBox" name="spin_green">
            <property name="maximum">
             <number>255</number>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_blue">
            <property name="text">
             <string>Blue</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="color_widgets::GradientSlider" name="slider_red">
            <property name="maximum">
             <number>255</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="4" column="2">
           <widget class="QSpinBox" name="spin_blue">
            <property name="maximum">
             <number>255</number>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="color_widgets::GradientSlider" name="slider_blue">
            <property name="maximum">
             <number>255</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tabHSV">
         <attribute name="title">
          <string>HSV</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="4" column="2">
           <widget class="QSpinBox" name="spin_value">
            <property name="maximum">
             <number>255</number>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="color_widgets::HueSlider" name="slider_hue">
            <property name="maximum">
             <number>359</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="color_widgets::GradientSlider" name="slider_saturation">
            <property name="maximum">
             <number>255</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="color_widgets::GradientSlider" name="slider_value">
            <property name="maximum">
             <number>255</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="QSpinBox" name="spin_saturation">
            <property name="maximum">
             <number>255</number>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QSpinBox" name="spin_hue">
            <property name="maximum">
             <number>359</number>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_hue">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Hue</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_saturation">
            <property name="text">
             <string>Saturation</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_value">
            <property name="text">
             <string>Value</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title">
         <string/>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="lable_alpha">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Alpha</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="color_widgets::GradientSlider" name="slider_alpha">
           <property name="maximum">
            <number>255</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spin_alpha">
           <property name="maximum">
            <number>255</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout_3">
        <property name="sizeConstraint">
         <enum>QLayout::SetMinimumSize</enum>
        </property>
        <property name="topMargin">
         <number>5</number>
        </property>
        <item row="3" column="0">
         <widget class="QLabel" name="label_hex">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Hex RGB</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_name">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Color Name</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0" colspan="3">
         <widget class="Line" name="line">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="4" column="1" colspan="2">
         <widget class="QLineEdit" name="name"/>
        </item>
        <item row="3" column="1" colspan="2">
         <widget class="color_widgets::ColorLineEdit" name="edit_hex"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>color_widgets::ColorWheel</class>
   <extends>QWidget</extends>
   <header>QtColorWidgets/color_wheel.hpp</header>
   <container>1</container>
   <slots>
    <signal>colorSelected(QColor)</signal>
    <slot>setColor(QColor)</slot>
   </slots>
  </customwidget>
  <customwidget>
   <class>color_widgets::GradientSlider</class>
   <extends>QSlider</extends>
   <header>QtColorWidgets/gradient_slider.hpp</header>
  </customwidget>
  <customwidget>
   <class>color_widgets::ColorLineEdit</class>
   <extends>QLineEdit</extends>
   <header>QtColorWidgets/color_line_edit.hpp</header>
  </customwidget>
  <customwidget>
   <class>color_widgets::HueSlider</class>
   <extends>QSlider</extends>
   <header>QtColorWidgets/hue_slider.hpp</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>spin_blue</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>removeButton</tabstop>
  <tabstop>spin_green</tabstop>
  <tabstop>tableView</tabstop>
  <tabstop>newButton</tabstop>
  <tabstop>spin_red</tabstop>
  <tabstop>slider_red</tabstop>
  <tabstop>slider_green</tabstop>
  <tabstop>slider_blue</tabstop>
  <tabstop>slider_hue</tabstop>
  <tabstop>slider_saturation</tabstop>
  <tabstop>slider_value</tabstop>
  <tabstop>spin_hue</tabstop>
  <tabstop>spin_saturation</tabstop>
  <tabstop>spin_value</tabstop>
 </tabstops>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>spin_red</sender>
   <signal>valueChanged(int)</signal>
   <receiver>slider_red</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>886</x>
     <y>282</y>
    </hint>
    <hint type="destinationlabel">
     <x>835</x>
     <y>280</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spin_green</sender>
   <signal>valueChanged(int)</signal>
   <receiver>slider_green</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>886</x>
     <y>311</y>
    </hint>
    <hint type="destinationlabel">
     <x>835</x>
     <y>312</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>slider_blue</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spin_blue</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>835</x>
     <y>344</y>
    </hint>
    <hint type="destinationlabel">
     <x>866</x>
     <y>329</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spin_alpha</sender>
   <signal>valueChanged(int)</signal>
   <receiver>slider_alpha</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>887</x>
     <y>429</y>
    </hint>
    <hint type="destinationlabel">
     <x>834</x>
     <y>425</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>slider_red</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spin_red</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>677</x>
     <y>271</y>
    </hint>
    <hint type="destinationlabel">
     <x>866</x>
     <y>271</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>slider_green</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spin_green</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>677</x>
     <y>300</y>
    </hint>
    <hint type="destinationlabel">
     <x>866</x>
     <y>300</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>slider_hue</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spin_hue</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>558</x>
     <y>263</y>
    </hint>
    <hint type="destinationlabel">
     <x>865</x>
     <y>260</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>slider_saturation</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spin_saturation</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>558</x>
     <y>295</y>
    </hint>
    <hint type="destinationlabel">
     <x>865</x>
     <y>292</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>slider_value</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spin_value</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>558</x>
     <y>327</y>
    </hint>
    <hint type="destinationlabel">
     <x>865</x>
     <y>324</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spin_hue</sender>
   <signal>valueChanged(int)</signal>
   <receiver>slider_hue</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>865</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>558</x>
     <y>263</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spin_saturation</sender>
   <signal>valueChanged(int)</signal>
   <receiver>slider_saturation</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>865</x>
     <y>292</y>
    </hint>
    <hint type="destinationlabel">
     <x>558</x>
     <y>295</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spin_value</sender>
   <signal>valueChanged(int)</signal>
   <receiver>slider_value</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>865</x>
     <y>324</y>
    </hint>
    <hint type="destinationlabel">
     <x>558</x>
     <y>327</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>slider_alpha</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spin_alpha</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>834</x>
     <y>425</y>
    </hint>
    <hint type="destinationlabel">
     <x>887</x>
     <y>429</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
